### ISSUE TITLE
error: ruby_nonempty_memcpy 

### ISSUE THREAD

**Issue by pawelwiewiora** [foxtacles/rj_schema#12]
There is an error while trying update from 1.0.2 to 1.0.3

### How to reproduce
```
$ docker run --rm -it ruby:2.7.2-alpine3.13 sh
# apk update && apk upgrade && \
   apk add --no-cache build-base && \
   gem update --system && gem install bundler
# gem install rj_schema -v '1.0.3'
```
### Error
```
compiling rj_schema.cpp
┃┃ rj_schema.cpp:19:10: error: no member named 'ruby_nonempty_memcpy' in the global namespace; did you mean simply 'ruby_nonempty_memcpy'?
┃┃                 return ::ruby_nonempty_memcpy(dest, src, n);
┃┃                        ^~~~~~~~~~~~~~~~~~~~~~
┃┃                        ruby_nonempty_memcpy
┃┃ rj_schema.cpp:18:22: note: 'ruby_nonempty_memcpy' declared here
┃┃         static inline void* ruby_nonempty_memcpy(void *dest, const void *src, size_t n) {
┃┃                             ^
┃┃ 1 error generated.
┃┃ make: *** [rj_schema.o] Error 1
```
### Additional information

1. Works on:
- Ubuntu/Debian
2. Fails on:
- Alpine and OSX

**Comment by foxtacles** [comment]
Thank you! It should be fixed in 1.0.4. I'm only able to verify that it works on Alpine now. Could you check OSX? (the GitHub test runner works)

**Comment by pawelwiewiora** [comment]
works on OSX, problem solved :)
